unity-renderer/Assets/DCLPlugins/BuilderInWorld/HUD/ProjectsPanelHUD/Scripts/BuilderMainPanelController.cs

```diff
@@ -55,12 +54,17 @@ public class BuilderMainPanelController : IHUD, IBuilderMainPanelController
 
     internal IContext context;
 
+    BaseVariable<Transform> configureBuilderInFullscreenMenu => DataStore.i.exploreV2.configureBuilderInFullscreenMenu;
```

Comment:
Do we need this? 

Is any situation where we won't show the panel in fullscreen?

Reply:
The Builder panel HUD must wait to receive the specific `Transform` (from the `ExploreV2` feature) where it has to be parented. This transform is not fully fullscreen but is part of a specific visual area below the menu header. In this waym the builder panel will be re-adapted to the correct place in the correct moment.